homeassistant/components/energyzero/const.py

```diff
@@ -12,5 +12,6 @@
 
 SERVICE_TYPE_DEVICE_NAMES = {
     "today_energy": "Energy market price",
```

Comment:
```suggestion
    "today_energy": "Energy market price of today",
```
It might be useful to also adjust this so that it is clear what is today or tomorrow, but I think it means that people will get a new service with entities and I don't know if that is really useful 🤔

Reply:
Good point, but as you concluded, that would lead to a breaking change, and I see this as an enhancement which shouldn't break anything